src/containers/social-links/social-links.styles.js

```diff
@@ -3,8 +3,8 @@ import { makeStyles } from '@material-ui/core/styles';
 export const useStyles = makeStyles(({ palette }) => ({
   cardBody: ({ position, fromSideBar }) => ({
     flex: position === 'center' ? 0 : 1,
-    padding: fromSideBar ? '70px 10px 0 3px' : '55px 10px 50px 40px',
-    margin: fromSideBar ? '35% 0 20% 0' : '',
+    padding: fromSideBar ? '50px 10px 0 3px' : '55px 10px 50px 40px',
+    margin: fromSideBar ? '30% 0 18% 0' : '',
```

Comment:
margins and paddings should not be in % value.

Reply:
fixed